### PR TITLE
[WIP] Responsive Design

### PR DESCRIPTION
**Pull request by beschoenen**
Example:
screenshot apr 17 2016 20-48-15

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
Finally someone who can fix that! =P Thanks

**Comment by miigotu** [comment]
@wimpyrbx  you might want to hold off on any gui changes until this is done, or it will conflict and be a bitch to get your changes in.

**Comment by miigotu** [comment]
Shit, caused you some conflicts, sorry.

**Comment by neoatomic** [comment]
Goed bezig, ;) :+1:

File: gui/slick/views/addShows_addExistingShow.mako [hunk 1]
```diff
@@ -4,53 +4,61 @@
...
+                                    <%include file="/inc_rootDirs.mako"/>
+					        </div>
+					        <div id="tabs-2" class="existingtabs">
+                                    <%include file="/inc_addShowOptions.mako"/>
+					        </div>
+				        </div>
+				        <br>
 
-    <p><b>${_('Displaying folders within these directories which aren\'t already added to SickRage')}:</b></p>
+				        <p>${_('''SickRage can add existing shows, using the current options, by using locally stored NFO/XML metadata to eliminate user interaction. If you would rather have SickRage prompt you to customize each show, then use the checkbox below.''')}</p>
```
Comment (miigotu) [inline gui/slick/views/addShows_addExistingShow.mako]: I would rather text not be changed in the same PR, this doesn't really make sense no in context.

Text/Code changes should be kept separate from layout changes.

Comment (miigotu) [inline gui/slick/views/addShows_addExistingShow.mako]: In fact, usually we want big PR like this broken up into smaller sections and merged in gradually nto make it easier to review. This PR is already almost impossible to review.

**Comment by beschoenen** [comment]
@miigotu I'll change those back.

And I'll split my PRs up in sections (show, config, etc). If you could merge this in a `responsive` branch or w/e would be great (if possible, otherwise I'll make a new pull request).

**Comment by miigotu** [comment]
I've created a responsive-ui branch. Just try and keep the scope to layout changes. You can make notes as you go of other things you think need changed (like certain pieces of translatable text, etc) and we can work that out afterwards or separate in develop.

I would like to get small pieces at a time merged into develop also, so that we aren't continually having to resolve conflicts.

**Comment by beschoenen** [comment]
@miigotu So this PR can be closed?

**Comment by miigotu** [comment]
@beschoenen yes, you can PR into the responsive-ui branch, or I can add you to the team so you can push directly to it if you like.

**Comment by miigotu** [comment]
I have added all except your last few changes into that branch already

**Comment by beschoenen** [comment]
I guess being in the team would be the most convenient.

**Comment by wimpyrbx** [comment]
@miigotu sure i'll hold off on any pull requests designwise until resolved.

**Pull request closed without merging**